src/main/java/com/google/sps/data/Utility.java

```diff
@@ -0,0 +1,234 @@
+package com.google.sps.data;
+
+import com.google.common.graph.*;
+import com.google.gson.Gson;
+import com.proto.MutationProtos.Mutation;
+import com.proto.MutationProtos.TokenMutation;
+import com.google.common.graph.EndpointPair;
+import java.util.HashMap;
+import java.util.List;
+import java.util.ArrayDeque;
+import java.util.ArrayList;
+import com.google.protobuf.Struct;
+import java.util.Map;
+import java.util.HashSet;
+import java.util.Set;
+import com.google.common.reflect.TypeToken;
+import java.lang.reflect.Type;
+import org.json.JSONObject;
+
+public class Utility {
```

Comment:
Make this final to avoid inheritance.

Also add a private constructor to enforce noninstantiability, since this class is supposed to be a "utility class" that carries a bunch of static methods.